editor/editor_command_palette.cpp

```diff
@@ -36,7 +36,6 @@
 #include "editor/gui/editor_toaster.h"
 #include "editor/themes/editor_scale.h"
 #include "scene/gui/control.h"
-#include "scene/gui/margin_container.h"
```

Comment:
I would keep those includes, so they're not just present by way of an implementation detail of PopupMenu, which may be changed back in the future.

Reply:
I meant all the `margin_container.h` includes which are being removed by this revert, also in other files.